e2e/search.spec.ts

```diff
@@ -133,7 +133,7 @@ test("Clearable checkbox filter works on desktop", async ({ page }) => {
   await expect(activeFilters).toContainText("Tuesday");
 
   const clearAll = page.locator("#active-filters-section button", {
-    hasText: "Clear",
+    hasText: "Clear Tuesday",
```

Comment:
i think we actually want this to test the "Clear all filters" button

Reply:
WHOOPS yes